server/routes/cohorts.js

```diff
@@ -121,4 +129,42 @@ router.get(
   })
 );
 
+router.patch(
+  '/:id',
+  asyncMiddleware(async (req, res) => {
+    const { email, user_id: userId, sub: localUserId } = req.user;
+    if (email && (userId || localUserId)) {
+      const { id } = req.params;
+      const { body } = req;
+      if (!id) {
+        return res.status(400).send({ message: 'Missing cohort id' });
+      }
+      // Get Cohort
+      const [cohort] = await getCohort(+id);
```

Comment:
What does the plus here do?

Reply:
converting string to number .... actually req.params. is key-value : string: string